### DISCUSSION TITLE
Accessability-Color_Blindness Friendly

### DISCUSSION THREAD

**Discussion by Gugatec** [bafolts/terraforming-mars#5416] [General]
Is there an "easy" way to change the colours of the players?

Even if I had to change the code, I would investigate how to do it, just need a hint to where to look/start.

Cheers.

**Comment by nwai90** [comment]
@Gugatec Check out https://github.com/nwai90/terraforming-mars/commit/29ff5ca144f5d8c38627b7447f677ab1aea8a9d5 for an example of how to modify player colours. Note you would have to update the two PNG graphics as well.

**Reply by Gugatec** [reply]
Hey, thanks for the fast reply, I will give it a try and share the results!

**Reply by Gugatec** [reply]
That was what I need to change the colors, right on the spot. Thank you @nwai90 !

**Comment by Gugatec** [comment]
It was actually trivial to change the colors.

I'm stuck now with something else, my environment to test and create is a MacBook air M1.

Everything is running ok in the workstation, if I create a container and upload it to docker, I can't run on my NAS that is intel based.

Now I need to create the image/container for amd64 platform.

I'm quite new to the docker and container, can someone shed some tips?

Cheers.

**Reply by Gugatec** [reply]
I figure it out, here is a summary of what I accomplished:

A friend has [Deuteranopia](https://www.color-blindness.com/deuteranopia-red-green-color-blindness/) and this is what he sees when playing (BTW, he is a very good player)

![image](https://user-images.githubusercontent.com/1827444/220338610-c8628551-f26a-4aaf-abd9-c66373142124.png)

He provided feedback on what colors would work, after some tweaking, this is what we figure it out:

Left column is what a normal person sees, the left is what Deuteranopia sees:

<img width="681" alt="image" src="https://user-images.githubusercontent.com/1827444/220338998-97c47a1e-e122-47a7-bca8-09dd9c92e6a9.png">

Here is an example of the creation page looks like now:

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/1827444/220339372-e0d84863-6a38-4ec8-a7b7-92b493bd2559.png">

Here is the [docker image](https://hub.docker.com/repository/docker/gugatec/terraforming-mars_deuteranopia-colors/general)

Cheers!


NOTE: Updated the docker image. Now The colony ships have the same colors. Turmoil is still missing, as I don't play that mode.

**Comment by kberg** [comment]
Recent updates have added icons on some of the player pieces.
